docs/source/contents/4_draft_mapping.rst

```diff
@@ -1331,6 +1331,297 @@ location
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+        <https://example.org/objects/1> edm:provider <http://id.loc.gov/authorities/names/n87808088> .
+
+Note that items held by UTK can also appear without a URI defining the institution as with `tdh:8803 <https://digital.lib.utk.edu/collections/islandora/object/tdh%3A8803/datastream/MODS/view>`_.
+It looks like we also have a misspelling here.
+
+.. code-block:: xml
+
+    <recordInfo>
+        <recordContentSource>University of Tennesse Knoxville. Libraries</recordContentSource>
+    </recordInfo>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+        <https://example.org/objects/1> edm:provider "University of Tennesse Knoxville. Libraries" .
+
+Sometimes when the resource comes from another institution, that institution name is placed in <recordContentSource>. For instance,
+here's an `example record - cdf:70 <https://digital.lib.utk.edu/collections/islandora/object/cdf%3A70/datastream/MODS/view>`_.
+Here it is also coupled with an "Intermediate Provider" note. McClung's Egypt collection is also treated similarly.
+
+.. code-block:: xml
+
+    <recordInfo>
+        <recordContentSource valueURI="http://id.loc.gov/authorities/names/no2017113530">Langston Hughes Library (Children's Defense Fund Haley Farm)</recordContentSource>
+    </recordInfo>
+    <note displayLabel="Intermediate Provider">University of Tennessee, Knoxville. Libraries</note>
+    <location>
+        <physicalLocation valueURI="http://id.loc.gov/authorities/names/no2017113530">Langston Hughes Library (Children's Defense Fund Haley Farm)</physicalLocation>
+    </location>
+
+For the purposes of DPLA, we only need the <recordContentSource> value and not also the <physicalLocation> value. Because
+these institutions are not directly contributing to DPLA, they are listed as an edm:dataProvider instead of an edm:provider.
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+        <https://example.org/objects/1> edm:dataProvider <http://id.loc.gov/authorities/names/no2017113530> .
+
+In addition, the TDH collection has null values. An example is `tdh:8597 <https://digital.lib.utk.edu/collections/islandora/object/tdh%3A8597/datastream/MODS/view>`_.
+This needs to be addressed before migration so that what UTK shares with DPLA meets its requirements.
+
+.. code-block:: xml
+
+    <recordInfo>
+        <recordContentSource/>
+    </recordInfo>
+
+Finally, there are 59 records that do not have <recordContentSource>. A few of these are starter namespaces (like baseball) that
+will be filled out once metadata for the collection is created. 55 of the records are associated with Kintner. This collection
+is currently being remediated by Andrew and this missing element will be addressed. Here's an `example record - kintner:10 <https://digital.lib.utk.edu/collections/islandora/object/kintner%3A10/datastream/MODS/view>`_.
+
```

Comment:
If the purpose of this is to deal with it before migration, is it even it the scope of the draft mapping?  If so, why is this here?

Reply:
Okay - I'll go ahead and delete this information.

Reply:
This was resolved in 2bd47b57b297b1805e1b9a7e6f517ce517eda056.